src/parser/asts/variables.c

```
@@ -0,0 +1,72 @@
+
+		node->value = t.value;
+		return node;
+	}
+
+	if(t.type == KEYWORD) {
+		AST_NODE* node = createASTNode(AST_VARIABLE_REFERENCE);
+		node->endingIndex = index + 1;
+		node->value = t.value;
+
+		return node;
+	}
```

cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)